### PR TITLE
MVVM + RxJava Common mistakes

### PR DESCRIPTION
**Pull request by luciapayo**
In progress

### CODE REVIEW

**Comment by florina-muntenescu** [comment]
for me it's good to merge. 
@pkrauss-asideas  - do you also want to check it out?

File: _posts/blog/2016-05-16-mvvm_rx_common_mistakes.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    @Override
+    public void onPause() {
+        mSubscription.clear();
+        super.onPause();
+    }
+}
+
+{% endhighlight %}
+
+We did exactly this and started receiving bug reports indicating a wrong end state in the ViewPager. Those types of bugs usually have the feared characteristic of not being 100% reproducible; sometimes they happen and sometimes they don’t, without any clear way to reproduce them. We had to face the brutal reality: we had race conditions, but why? We have a neat architecture using MVVM, everything is unit tested and we use RxJava to send events.
```
Comment (ghost) [inline _posts/blog/2016-05-16-mvvm_rx_common_mistakes.md]: a bit often reproducible

**Comment by ghost** [comment]
👍 great reading - thanks for taking the effort @luciapayo !!!

File: _posts/blog/2016-05-16-mvvm_rx_common_mistakes.md (hunk 1 above)
Comment (luciapayo) [inline _posts/blog/2016-05-16-mvvm_rx_common_mistakes.md]: What do you mean? I don't see what you are refering to

Comment (florina-muntenescu) [inline _posts/blog/2016-05-16-mvvm_rx_common_mistakes.md]: the word "reproducible" appears too much.

**Comment by luciapayo** [comment]
Thanks @pkrauss-asideas and @florina-muntenescu for the review!! I will merge now :)

**Pull request merged**